### PR TITLE
FireEye CM/AM Appliance Integration

### PR DESCRIPTION
**Pull request by jfryman**
# FireEye Integration Pack

StackStorm integration with FireEye CM and AX threat intelligence devices

## Configuration

* `api_url` - HTTPS endpoint of FireEye CM appliance. (e.x.: https://fqdn.to.device)
* `username` - FireEye username
* `password` - FireEye password

## Actions

* `fireeye.get_alert_query`         - Request existing alert profiles with optional filters
* `fireeye.get_submission_results`  - Query results of completed job
* `fireeye.get_submission_status`   - Query status of running job
* `fireeye.submit_malware`          - Submit a Malware object to FireEye AX appliance
* `fireeye.view_ax_config`          - Returns a list of profiles and applications on AX devices

### CODE REVIEW

File: packs/fireeye/actions/submit_malware.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from lib.actions import BaseAction
+import json
+import os
+
+
+class SubmitMalwareAction(BaseAction):
+    def run(self, **kwargs):
```
Comment (Kami) [inline packs/fireeye/actions/submit_malware.py]: Looking at the action metadata file, it also defines `file_path` and `file_type` parameters.

I assume we don't want to send those in the payload? Or it doesn't matter and it still works even if those parameters are included?

File: packs/fireeye/actions/submit_malware.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        payload = json.dumps(kwargs)
+        file_basename = os.path.basename(kwargs['file_name'])
```
Comment (Kami) [inline packs/fireeye/actions/submit_malware.py]: Looks like metadata only defines `file_path` and not `file_name` parameter.

File: packs/fireeye/actions/submit_malware.py (hunk 1 above)
Comment (jfryman) [inline packs/fireeye/actions/submit_malware.py]: Yes sir. Those do need to be in the payload. They are set as mandatory in the action metadata.

File: packs/fireeye/actions/submit_malware.py (hunk 2 above)
Comment (jfryman) [inline packs/fireeye/actions/submit_malware.py]: Good :eyes:. Updated the metadata.

**Pull request merged**